plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -59,28 +59,29 @@ def associate(source_repo, dest_repo, import_conduit, config, units=None):
     group_ids, rpm_names, rpm_search_dicts = identify_children_to_copy(associated_units)
 
     # ------ get group children of the categories ------
-    group_criteria = UnitAssociationCriteria([models.PackageGroup.TYPE],
-                                             unit_filters={'id': {'$in': list(group_ids)}})
-    group_units = list(import_conduit.get_source_units(group_criteria))
-    if group_units:
-        associated_units |= set(
-            associate(source_repo, dest_repo, import_conduit, config, group_units))
+    for page in paginate(group_ids):
```

Comment:
Minor point (and just my opinion).  The name `page` seems odd here.  suggest something like `batch`.